DOCKER.md

```diff
@@ -2,14 +2,14 @@
 
 ## Introduction
 
-Docker provides a container based mechanism that ensures consistency and portability as well as environment isolation. For these reasons, deployment of Vectara \Answer via Docker may be preferred in some cases.<br>
+Docker provides a container-based mechanism that ensures consistency and portability as well as environment isolation. For these reasons, deployment of Vectara \Answer via Docker may be preferred in some cases.<br>
 To demonstrate how to use Vectara Answer with Docker we will now walk through the steps for building an application with the content of Paul Graham's essays website (http://paulgraham.com/). 
 
 We call this application `AskPG`.
 
 ## Pre-Requisites
 
-For Docker you will first need to follow some additional installation steps:
+For Docker, you will first need to follow some additional installation steps:
 1. Install [Docker](https://docs.docker.com/engine/install/).
```

Comment:
Suggest adding here "For mac users, please ensure you Docker Desktop is version 4.26.1  or above"

Reply:
Okay. I'll add and resolve this conversation.
Thank you for your review!